### DISCUSSION TITLE
Minecraft Console Client now avaliable on Android with Termux

### DISCUSSION THREAD

**Discussion by milutinke** [ORelio/Minecraft-Console-Client#2160] [Announcements]
Greetings everyone!

![MCC <3 Android](http://i.pics.rs/RR63M.png)

Now it is possible to run the **Minecraft Console Client** on **Android** phones using **Termux** and **Ubuntu 22.04**.
If you are interested in it, take a look at our [documentation page](https://mccteam.github.io/docs/guide/installation.html#running-on-android) on how to install and run the MCC on Android phones.

*In the future, we are looking forward at porting the project to iOS and Android phones as a standalone app using Xamarin.
Note that this will take some time and a lot of work.*

**Comment by Splarkszter** [comment]
Tested. Works flawlessly.
